piccolo/apps/schema/commands/generate.py

```diff
@@ -107,31 +108,25 @@ def is_foreign_key(self, column_name: str) -> bool:
                 return True
         return False
 
-    def get_foreign_key_constraint_name(self, column_name) -> str:
+    def get_foreign_key_constraint_name(
+        self, column_name
+    ) -> t.Tuple[str, str] | ValueError:  # type: ignore
         for i in self.foreign_key_constraints:
             if i.column_name == column_name:
-                return i.constraint_name
+                return i.constraint_name, i.constraint_schema
```

Comment:
How about returning a dataclass instead of a tuple. Something like:

```
@dataclass
class Constraint:
    name: str
    schema: str
```

I find them a bit nicer than tuples, but up to you.

Reply:
Very good idea. I'll definitely change this.